CONTRIBUTING.md

```diff
@@ -20,6 +20,20 @@ We try to stick with Google's code standards. Please call us on them if we don't
 3. You may merge the Pull Request in once you have the sign-off of two other developers, or if you
    do not have permission to do that, you may request the second reviewer to merge it for you.
 
+## Issue Labels  
+If you are a contributor to this repository it is your responsibility to make sure all issues are properly labeled. We are following [this](https://robinpowered.com/blog/best-practice-system-for-organizing-and-tagging-github-issues/) standard. As follows,
```

Comment:
I wouldn't say we all following their standard so much as our standard was inspired by.. that way we can modify as we please